src/main/scala/no/ndla/listingapi/service/package.scala

```diff
@@ -0,0 +1,10 @@
+package no.ndla.listingapi
+
+package object service {
+  def createOembedUrl(oldNodeId: Option[Long]): Option[String] = {
```

Comment:
Jeg er ingen fan av å ha `Option` som parameter i en funksjon. Det fører ofte til ulogisk kodeflyt, og det er ikke intuitivt hva `Option[Long]` faktisk betyr. Bedre bare å ha en `Long` som parameter. Evt. en `Option` som output parameter også, men her er en plain `String` bedre.

I stedet for å gjøre `createOembedUrl(cover.oldNodeId)` blir det i stedet `cover.oldNodeId.map(createOembedUrl)`